runtime/vm/vmprops.c

```diff
@@ -1433,3 +1444,644 @@ getMUtf8String(J9JavaVM *vm, const char *userString, UDATA stringLength)
 	}
 	return result;
 }
+
+/**
+ * @return 0 on success, -1 on error
+ */
+static jint
```

Comment:
This is used as if it returned a boolean value, suggest the return type should instead be `jboolean`.